centreon/tests/e2e/cypress/e2e/Resources-status/02-acknowledgments/index.ts

```diff
@@ -0,0 +1,649 @@
+      .parent()
+      .parent()
+      .getByLabel({ label: `${serviceInAcknowledgementName} Acknowledged` })
+      .trigger('mouseover');
+
+    cy.get('div[role="tooltip"]').should('be.visible');
+
+    cy.contains(hostInAcknowledgementName)
+      .parent()
+      .parent()
+      .getByLabel({ label: `${hostInAcknowledgementName} Acknowledged` })
+      .trigger('mouseover');
+
+    cy.get('div[role="tooltip"]').should('be.visible');
+
+    tearDownResource().then(() => tearDownAckResource());
+  }
+);
+
+Given('the "Resource Problems" filter enabled', () => {
+  cy.contains('Unhandled alerts');
+});
+
+Given('criteria is {string}', (criteria: string) => {
+  typeToSearchInput(criteria);
+});
+
+Given(
+  'a resource of host is selected with {string}',
+  (initial_status: string) => {
+    const hostStatus = {
+      host: hostChildInAcknowledgementName,
+      output: `submit_${hostChildInAcknowledgementName}_${initial_status}`,
+      status: initial_status
+    };
+
+    submitCustomResultsViaClapi(hostStatus);
+
+    checkThatFixtureHostsExistInDatabase(
+      hostChildInAcknowledgementName,
+      `submit_${hostChildInAcknowledgementName}_${initial_status}`,
+      [hostStatus]
+    );
+
+    cy.refreshListing();
+
+    cy.contains(hostChildInAcknowledgementName)
+      .parent()
+      .parent()
+      .find('input[type="checkbox"]:first')
+      .click();
+  }
+);
+
+Given(
+  'a resource of service is selected with {string}',
+  (initial_status: string) => {
+    const serviceStatus = {
+      host: hostInAcknowledgementName,
+      output: `submit_${serviceInAcknowledgementName}_${initial_status}`,
+      service: serviceInAcknowledgementName,
+      status: initial_status
+    };
+
+    submitCustomResultsViaClapi(serviceStatus);
+
+    checkThatFixtureServicesExistInDatabase(
+      serviceInAcknowledgementName,
+      `submit_${serviceInAcknowledgementName}_${initial_status}`,
+      [serviceStatus]
+    );
+
+    cy.refreshListing();
+
+    cy.contains(serviceInAcknowledgementName)
+      .parent()
+      .parent()
+      .find('input[type="checkbox"]:first')
+      .click();
+  }
+);
+
+When('"sticky" checkbox is {string} in the form', (check: string) => {
+  switch (check) {
+    case 'unchecked':
+      cy.getByLabel({ label: 'Sticky' }).uncheck();
+      cy.getByLabel({ label: 'Sticky' }).should('not.be.checked');
+      break;
+    default:
+      cy.getByLabel({ label: 'Sticky' }).check();
+      cy.getByLabel({ label: 'Sticky' }).should('be.checked');
+      break;
+  }
+});
+
+When('"persistent" checkbox is {string} in the form', (check: string) => {
+  switch (check) {
+    case 'unchecked':
+      cy.getByLabel({ label: 'Persistent' }).uncheck();
+
+      cy.getByLabel({ label: 'Persistent' }).should('not.be.checked');
+      break;
+    default:
+      cy.getByLabel({ label: 'Persistent' }).check();
+
+      cy.getByLabel({ label: 'Persistent' }).should('be.checked');
+      break;
+  }
+});
```

Comment:
I think we can remove the persistent related stuff because the PR that removes the persistent checkbox merged in the next hours